wp-content/themes/thinkery/style.css

```diff
@@ -1485,23 +1486,68 @@ p.entry-meta {
 	margin-bottom: 0;
 }
 
+.footer-widgets div ul li a {
+	color: white;
+	font-family: museo-sans, sans-serif;
+	font-size: .85em;
+	font-weight: 400;
+	text-decoration: none;
+}
+.footer-widgets #fa-icons {
+display: flex;
+justify-content: space-between;
+font-size: 1.5em;
+
+}
+
+.footer-widgets #fa-icons i:focus,
+.footer-widgets #fa-icons i:hover {
+	color: #1b2432;
+}
```

Comment:
A lot of these color styles around the footer social icons are probably unnecessary, if we just do the following, it should cascade down through all footer links, icon or not.
```
.footer-widgets a {
    color: #fff;
}
.footer-widgets a:hover, 
.footer-widgets a:focus, {
    color: #1b2432;
}
```
*Note I didn't see social in the testing notes so I added my own for testing, noting below in case we had a difference there? 
```
<div id="fa-icons">
      <a target="_blank" href="https://twitter.com/Thinkeryatx"><i class="fab fa-twitter"></i></a>
      <a target="_blank" href="http://facebook.com/TheThinkery"><i class="fab fa-facebook"></i></a>
      <a target="_blank" href="http://www.pinterest.com/ThinkeryATX/"><i class="fab fa-pinterest"></i></a>
      <a target="_blank" href="http://www.youtube.com/user/thinkeryaustin/"><i class="fab fa-youtube"></i></a>
      <a target="_blank" href="http://www.instagram.com/thinkeryatx/"><i class="fab fa-instagram"></i></a>
</div>
```